README.md

```diff
@@ -19,7 +19,7 @@ $ source $HOME/.cargo/env
 $ rustup component add rustfmt
 ```
 
-If your rustc version is lower than 1.39.0, please update it:
+If your rustc version is lower than 1.45.0, please update it:
```

Comment:
```suggestion
Please sure you are always using the latest stale rust version by running:
```

Reply:
Type: 'stale'. Can't find a way to suggest against the suggestion, so I did a normal commit instead

Reply:
lol thanks :). yeah not a stale version